### PR TITLE
upgrade to latest babelify

### PR DESCRIPTION
**Pull request by Bjvanminnen**
Upgrade to a newer babelify, that uses babel v6. This appears to work on my version of node (v0.12.7).

### CODE REVIEW

File: apps/Gruntfile.js [hunk 1]
```diff
@@ -297,7 +297,7 @@ module.exports = function (grunt) {
       cmd += ' --cachefile ' + outputDir + options.cacheFile;
     }
     cmd += ' --extension=.jsx' +
-      ' -t [ babelify --compact=false --sourceMap --sourceMapRelative="$PWD" ]' +
+      ' -t [ babelify --compact=false --sourceMap ]' +
```
Comment (Bjvanminnen) [inline apps/Gruntfile.js:300]: Based on a comment in babelify, sourceMapRelative became the default in babel v6.0 and this option is now ignored.

File: apps/src/applab/ChartApi.js [hunk 2]
```diff
@@ -10,7 +10,7 @@
 var utils = require('../utils');
 var AppStorage = require('./appStorage');
 var GoogleChart = require('./GoogleChart');
-require("babelify/polyfill"); // required for Promises in IE / Phantom
+require("babel-polyfill"); // required for Promises in IE / Phantom
```
Comment (Bjvanminnen) [inline apps/src/applab/ChartApi.js:13]: we may want to move where we require this, but im going to leave that for a separate PR

File: apps/.babelrc [hunk 3]
```diff
@@ -0,0 +1,6 @@
+{
+  "presets": ["es2015", "react" ],
+  "plugins": [
+    "add-module-exports"
```
Comment (Bjvanminnen) [inline apps/.babelrc:4]: Plugin is needed to be able to have

```
// foo.js
export default function foo() {}

// bar.js
var foo = require('./foo');
```

**Comment by Bjvanminnen** [comment]
Might be a bad idea to commit right before facilitator summit. Probably makes sense to just wait until Monday.

**Comment by islemaster** [comment]
```
  1673 passing (2m)
```

`npm test` passed for me locally.  Nice work!

**Comment by Bjvanminnen** [comment]
Going to reopen against staging-next
